### DISCUSSION TITLE
Exit code 127 - Command Cues

### DISCUSSION THREAD

**Discussion by nickmuzzy** [FrancescoCeruti/linux-show-player#308] [Q&A] [answered]
Hello all,

I'm having problems with command cues. I'm sure it's because I don't know what I'm doing. 

Whatever input I try, including the commands suggested in the wiki, I get the same error message:

"Command cue ended with an error status. Exit code: 127."

I'd very much appreciate any help anyone can offer.

Thanks!

**Comment by FrancescoCeruti** [comment]
Any command?

If I'm not mistaken it indicates that some commands cannot be found.

Are you using the flatpak version?

**Comment by nickmuzzy** [comment]
Hi, 

Yes, I've been using the flatpack version. Haven't been able to make anything work...

N

**Reply by FrancescoCeruti** [reply, marked as answer]
See #310 for a workaround

**Comment by nickmuzzy** [comment]
All working now - many thanks!
